src/app/campaign/inventory/goal-form.component.ts

```diff
@@ -98,24 +103,29 @@ export class GoalFormComponent implements OnInit {
 
   ngOnInit() {
     this.goalForm = this.formController.control as FormGroup;
+    this.velocitySub = this.goalForm.get('velocity').valueChanges.subscribe(velocity => this.setDailyMinimum(velocity));
+    this.goalMinSub = combineLatest([this.goalForm.get('totalGoal').valueChanges, this.goalForm.get('dailyMinimum').valueChanges])
```

Comment:
once totalGoal and dailyMinimum have been set, the velocity will be set accordingly